### ISSUE TITLE
How to active the GPU (cuda)?

### ISSUE THREAD

**Issue by jesusbft** [mittagessen/party#5]
I'm trying to test the party train. The command only use the CPU, not GPU.

look like the variable ctx.meta['device'] is setup as a cpu, not cuda. I couldn't find how the context working here.

    try:
        accelerator, device = to_ptl_device(ctx.meta['device'])
    except Exception as e:
        raise click.BadOptionUsage('device', str(e))

party train --load-from-repo 10.5281/zenodo.14616981 -t dataset/train.lst -e dataset/val.lst -B 16 --workers 16 --threads 16 -o models/Party_TrPraInq -q early --lag 20 --augment

_**cpu**_    (I use the print(device) to confirm it

GPU available: True (cuda), used: False
TPU available: False, using: 0 TPU cores
HPU available: False, using: 0 HPUs
`Trainer(val_check_interval=1.0)` was configured so validation will run at the end of the training epoch..
Loading from huggingface hub 10.5281/zenodo.14616981.
┏━━━┳━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━┓
┃   ┃ Name            ┃ Type             ┃ Params ┃ Mode  ┃
┡━━━╇━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━┩
│ 0 │ model           │ OptimizedModule  │  214 M │ train │
│ 1 │ model._orig_mod │ PartyModel       │  214 M │ train │
│ 2 │ criterion       │ CrossEntropyLoss │      0 │ train │
│ 3 │ val_mean        │ MeanMetric       │      0 │ train │
└───┴─────────────────┴──────────────────┴────────┴───────┘
Trainable params: 214 M
Non-trainable params: 0
Total params: 214 M
Total estimated model params size (MB): 856
Modules in train mode: 1078
Modules in eval mode: 0

**Comment by jesusbft** [comment]
Found the option, sorry:

 party -d cuda:0 train
